### DISCUSSION TITLE
Regional prompting/Latent couple/Area composition

### DISCUSSION THREAD

**Discussion by aolko** [lllyasviel/Fooocus#913] [Ideas]
It would be nice to let people specify custom regions and associate prompts with them (A1111 and ComfyUI have this)
![](https://github.com/ashen-sensored/stable-diffusion-webui-two-shot/blob/main/screenshots/20230303.png?raw=true)

https://youtu.be/uR89wZMXiJ8

**Comment by ivancollider** [comment]
If this were introduced, it would be a game changer, it would make fooocus even more precise, there would be no more need to go crazy combining masks together, it’s one of the features I’m looking forward to the most.”

**Comment by aolko** [comment]
This feature recently went live in invoke:
https://youtu.be/CLVylJAMIF8
Please implement it.

**Comment by E2GO** [comment]
+1 for implementation - very useful technique for complex compositions.

**Comment by Nerual-Dreming** [comment]
friends, this is a useless feature, you will spend exactly the same amount of time on inpainting as if you just do the inpainting right away. It doesn't give good quality. Just try using it separately, it only looks cool.

**Reply by ILikeToasters** [reply]
Not sure it is useless but I do agree it is not as big of a deal as many make it out to be so far in my experience. I would like to see it as another option but there are other things that would be more useful.

**Comment by xhoxye** [comment]
It simply combines the three Generates of three different masks into one Generate
